README.md

```diff
@@ -1 +1,33 @@
-Hello
+# moves-like-swagger-api
+
+Database
+---
+1. Run each SQL script in order. 
```

Comment:
Lets specify in numbered order, just in case the order of the files ever changes.